_posts/0000-01-02-Cyb3rm4n20.md

```diff
@@ -0,0 +1,6 @@
+< ---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
It's very easy to make some words **bold** and other words *italic* with Markdown. You can even [link to Google!](http://google.com) #3

Reply:
yes it is